reference/constraints/Choice.rst

```diff
@@ -279,6 +279,35 @@ you can pass the class name and the method as an array.
             }
         }
 
+Supplying the Choices from an Array Constant
```

Comment:
array constant in lower case is better i think, wdyt?

Reply:
yes you're right

Reply:
actually i've wrote like this, to be consistent with the title above which is `Supplying the Choices with a Callback Function`